application/common/models/Method.php

```diff
@@ -364,11 +368,13 @@ public static function deleteExpiredUnverifiedMethods()
                                 ->all();
 
         foreach ($methods as $method) {
-            if ( ! $method->delete()) {
+            try {
+                $method->delete();
```

Comment:
It'll return the number of rows affected or an exception if there was an error. It is possible to return zero, but if it cant delete for some reason an exception should be thrown.